src/components/SettingsDialog/ExportBoard/ExportHintHiddenContent/ExportHintHiddenContent.tsx

```diff
@@ -0,0 +1,40 @@
+import {useTranslation} from "react-i18next";
+import {ReactComponent as InfoIcon} from "assets/icon-warning.svg";
+import {useAppSelector} from "store";
+import {Column} from "../../../../types/column";
+import "./ExportHintHiddenContent.scss";
+
+export interface ExportHintHiddenContentProps {
+  columns: Column[];
+}
+
+const ExportHintHiddenContent = ({columns}: ExportHintHiddenContentProps) => {
+  const {t} = useTranslation();
+
+  const hiddenColumns = columns.filter((col) => !col.visible);
+  const showNotesOfOtherParticipants = useAppSelector((state) => state.board.data?.showNotesOfOtherUsers);
```

Comment:
Instead of receiving all columns via props, you could just get them from the store in this component. 
The parent component only gets the columns from the store to pass them to this component.

Reply:
Thanks for noticing that, I just copied the existing code. This makes it way cleaner.